spec/std/isa/csr/H/henvcfg.yaml

```
@@ -186,6 +187,37 @@ fields:
       return (implemented?(ExtensionName::Svadu)) ? CsrFieldType::RO : CsrFieldType::RW;
     reset_value(): |
       return (implemented?(ExtensionName::Svadu)) ? UNDEFINED_LEGAL : 0;
+    sw_write(csr_value): |
+      if (CSR[menvcfg].ADUE == 1'b0) {
+        return 0;
+      }
+      return csr_value.ADUE;
+  DTE:
+    location: 59
+    description: |
+      *Double Trap Enable*
+
```

Let's make that the "long_name":
```suggestion
    long_name: Double Trap Enable
    description: |
```

---

Hm sure , ok sir I thought it before because of that my Sstc PR where sir @dhower-qc also told the same to do, but still didn't make it out but will definitely do by today/tomorrow.